packages/api-kit/src/types/safeTransactionServiceTypes.ts

```diff
@@ -3,6 +3,7 @@ import {
   SafeMultisigTransactionResponse,
   SafeTransactionData
 } from '@safe-global/safe-core-sdk-types'
+import SafeOperation from '@safe-global/relay-kit/src/packs/safe-4337/SafeOperation'
```

Comment:
Here we are consumin a class from another kit that we are not importing in the dependencies. We should minimize cross-package dependencies so it's likely we need the type to be defined in `safe-core-sdk-types`

Reply:
Updated accordingly and fixed tests as well. Ready to be reviewed again